src/core/logger/logger.test.ts

```
@@ -1,45 +1,130 @@
-import { describe, it, expect, vi, beforeEach } from 'vitest';
-import { Logger, logger } from '../logger/index.js';
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest';
+import { Logger } from '../logger/index.js';
+import * as fs from 'fs';
+import * as path from 'path';
+import { tmpdir } from 'os';
 
 // Ensure console.log is mocked and environment is reset between tests
 beforeEach(() => {
     delete process.env.SAIKI_LOG_LEVEL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)